### PR TITLE
[ES] added missing sections + semantic line breaks in content/es/style-guide & content/es/contribute

### PR DESCRIPTION
**Pull request by marianogg9**
### Describe your changes
- added missing sections + semantic line breaks in `content/es/style-guide/_index.md`.
- added missing section in `content/es/contribute/_index.md`.

### Related issue number or link (ex: `resolves #issue-number`)
- #1667 

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/style-guide/_index.md [hunk 1]
```diff
@@ -24,7 +25,16 @@ El Glosario Cloud Native sigue las [guías de estilo por defecto](https://github
 
 ## Audiencia
 
-El Glosario está escrito para una audiencia técnica Y no técnica. Asegúrese de que las definiciones sean explicadas en términos simples y no asuma conocimientos técnicos. Más acerca de eso a continuación en Definición.
+El Glosario está escrito para una audiencia técnica y no técnica.
+Asegúrese de que las definiciones sean explicadas en términos simples y no asuma conocimientos técnicos. Más acerca de eso a continuación en Definición.
+
+## Contenido mínimo viable
+Nuestro objetivo es facilitar la comprensión de términos nativos de nube a cualquiera.
```
Comment (electrocucaracha) [inline content/es/style-guide/_index.md:32]:
```suggestion
## Contenido mínimo viable

Nuestro objetivo es facilitar la comprensión de términos nativos de nube a cualquiera.
```

File: content/es/style-guide/_index.md [hunk 2]
```diff
@@ -100,31 +110,74 @@ category: Concepto
...
+
+
+**Haga su investigación**: Incluso si usted se siente confiado de que conoce el término, verifique que realmente es como lo piensa.
+Habitualmente utilizamos términos en una organización de una cierta manera que no refleja el panorama completo.
+Cuando haga su investigación, especialmente cuando el término no sea del todo familiar, utilice múltiples fuentes.
+Muchas definiciones son algo subjetivas, especialmente si provienen de un proveedor.
+El Glosario debe contener definiciones aceptadas globalmente y sin perspectivas particulares.
+
+
+**No cometa plagio**. Como cualquier publicación seria, las mismas reglas aplican al Glosario.
```
Comment (electrocucaracha) [inline content/es/style-guide/_index.md:138]:
```suggestion
Tenga en cuenta que la etiqueta **properties** no requiere secciones separadas.Una definición será suficiente.

Para facilitar la revisión, por favor utilice **[semántica de saltos de línea](https://sembr.org/)** (una oración por línea).

#### La calidad es esencial

Si ha sido agregada, su contribución sera la definición oficial de la CNCF para ese término (hasta que alguien más la mejore).
Crear un término que cumpla con los altos estándares de la CNCF no puede ser un proceso apurado - la calidad toma su tiempo y esfuerzo.

**Haga su investigación**: Incluso si usted se siente confiado de que conoce el término, verifique que realmente es como lo piensa.
Habitualmente utilizamos términos en una organización de una cierta manera que no refleja el panorama completo.
Cuando haga su investigación, especialmente cuando el término no sea del todo familiar, utilice múltiples fuentes.
Muchas definiciones son algo subjetivas, especialmente si provienen de un proveedor.
El Glosario debe contener definiciones aceptadas globalmente y sin perspectivas particulares.

**No cometa plagio**. Como cualquier publicación seria, las mismas reglas aplican al Glosario.
```

File: content/es/style-guide/_index.md [hunk 3]
```diff
@@ -100,31 +110,74 @@ category: Concepto
...
-Recomendamos comenzar con un documento de Google o Word, déjalo pasar durante unos días y volver a visitarlo. Esto te permitirá captar frases o expresiones que podrían redactarse de una forma más sencilla y accesible. Además, asegúrese de ejecutar un corrector ortográfico antes de enviar un PR.
+Recomendamos comenzar con un documento de Google o Word, déjalo pasar durante unos días y volver a visitarlo.
+Esto te permitirá captar frases o expresiones que podrían redactarse de una forma más sencilla y accesible.
+Además, asegúrese de ejecutar un corrector ortográfico antes de enviar un PR.
 
-Para asegurarse de que nadie más envíe un PR mientras trabaja en un término, asegúrese de reclamar un issue (o crear uno) y que se le asigne. Más sobre eso en el documento [Cómo contribuir](/es/contribute/).
+Para asegurarse de que nadie más envíe un PR mientras trabaja en un término, asegúrese de reclamar un issue (o crear uno) y que se le asigne.
+Más sobre eso en el documento [Cómo contribuir](/es/contribute/).
 
 Antes de comenzar, lea algunos de los términos publicados del Glosario para tener una idea del nivel de detalle y dificultad y cuándo los ejemplos son apropiados.
```
Comment (electrocucaracha) [inline content/es/style-guide/_index.md:170]:
```suggestion
Antes de comenzar, lea algunos de los términos publicados del Glosario
para tener una idea del nivel de detalle y dificultad y cuándo los ejemplos son apropiados.
```

File: content/es/style-guide/_index.md [hunk 4]
```diff
@@ -100,31 +110,74 @@ category: Concepto
...
 
 ## El proceso de revisión: qué esperar
 
-Tenga en cuenta que actualmente solo hay tres mantenedores haciendo esto en su tiempo libre. Ocasionalmente, podremos revisar los términos rápidamente; en otras ocasiones, puede llevar algo de tiempo. Agradecemos su paciencia. Si tiene alguna pregunta, comuníquese con nosotros en el canal #glossary de Slack (para saber dónde y cómo encontrarlo, consulte nuestro documento de [Cómo contribuir](/es/contribute/)).
+Tenga en cuenta que actualmente solo hay tres mantenedores haciendo esto en su tiempo libre.
+Ocasionalmente, podremos revisar los términos rápidamente; en otras ocasiones, puede llevar algo de tiempo.
+Agradecemos su paciencia.
```
Comment (electrocucaracha) [inline content/es/style-guide/_index.md:176]:
```suggestion
Ocasionalmente, podremos revisar los términos rápidamente; en otras ocasiones, puede llevar algo de tiempo — agradecemos su paciencia.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
@marianogg9 muchas gracias por la actualización de la guía. Deje algunos cambios mínimos al respecto. Además, parece ser que la sección de `Categoría` [fue](https://github.com/cncf/glossary/commit/fd7222d9d21e10dfda9142636df1119c7976da18) remplazada por `Etiquetas`, podrías agregar ese cambio?

File: content/es/style-guide/_index.md [hunk 5]
```diff
@@ -77,19 +88,30 @@ title: Plantilla de Definición
...
+Exceptuando "fundamental", que simplemente indica que el término referido es vital para entender otros conceptos de Cloud Native, la mayoría de los términos utilizara solo una etiqueta.
 
-La etiqueta de **category** vendrá después de la etiqueta de estado. Su valor debe ser uno de los siguientes valores:
+**Nota**: Por favor no cree nuevas etiquetas a menos que sean aprobadas por los mantenedores. Cuando agregue etiquetas, asegúrese de que estén correctamente deletreadas como se lista a continuación (en singular, sin errores).
 
-- Tecnología
-- Propiedad
-- Concepto
+Las etiquetas existentes son:
+- application
```
Comment (electrocucaracha) [inline content/es/style-guide/_index.md:101]:
```suggestion
Las etiquetas existentes son:

- application
```

**Review by electrocucaracha — APPROVED** [review]

**Review by ramrodo — APPROVED** [review]
/lgtm

**Pull request merged**